trio/_core/_ki.py

```diff
@@ -3,17 +3,21 @@
 import inspect
 import signal
 import sys
+import types
 from functools import wraps
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, Callable, Final, TypeVar
```

Comment:
I feel like it is better practice to `import Callable from collections.abc` instead of from typing, as the typing version is just an alias

Reply:
We've had this discussion in other PRs, and mostly settled on that it doesn't really matter and once we drop older versions and run `pyupgrade` it'll get autofixed.